src/main/java/com/atlassian/jira/cloud/jenkins/common/client/JiraApi.java

```diff
@@ -83,71 +85,49 @@ private void checkForErrorResponse(final String jiraSiteUrl, final Response resp
         if (!response.isSuccessful()) {
             final String message =
                     String.format(
-                            "Error response code %d when submitting builds for %s",
+                            "Error response code %d when submitting to %s",
                             response.code(), jiraSiteUrl);
             final ResponseBody responseBody = response.body();
             if (responseBody != null) {
                 log.error(
                         String.format(
-                                "Error response body when submitting builds for %s: %s",
+                                "Error response body when submitting to %s: %s",
                                 jiraSiteUrl, responseBody));
```

Comment:
The error message is not properly resolved. In order to get the error message from the `ResponseBody` we need to call `responseBody.string()`

Reply:
Fixed now.